src/families/tron/test-dataset.js

```diff
@@ -705,6 +705,7 @@ const dataset: DatasetTest<Transaction> = {
               To re-enable when the support will be done.
             */
             "balance is sum of ops",
+            "account always has tronResources",
```

Comment:
Does this test still belong in FIXME?

Reply:
I actually have no idea, I tried adding a `tests` property in the dataset, but it was falling if I remember correctly, so I just moved with the flow here without really knowing what the FIXME was for 😛 

What would be the actual good way to declare the test ?

Reply:
Yeah I agree this can be pretty confusing :sweat_smile: 
The `FIXME_` prefixed sections are actually used to bypass things. Here, the `FIXME_tests` section belongs to an account, and currently says that the generic `"balance is sum of ops" `test shouldn't be executed for this account.
Adding your new test here will do the same, I'm guessing this is not what you wanted?

Reply:
However, it just hit me that you kinda added a currency-specific test with the generic ones.
I didn't run your new tests, but doesn't it fail for currencies other than `tron`?
e.g. if you run: `yarn test --testNamePattern "account always has tronResources"`.
I think the way to do it is to "just" move this test to the `test-specifics.js` file in the tron family.

Reply:
> Adding your new test here will do the same, I'm guessing this is not what you wanted?

Yes it is, I need to test again doing this, cause from what I can remember it didn't work for whatever reason... 

> However, it just hit me that you kinda added a currency-specific test with the generic ones.
I didn't run your new tests, but doesn't it fail for currencies other than tron?

Actually I followed the insights of @gre just above, but I might have misunderstood how the `makeTest` is working. To me it felt like I was creating a test that would then be addable to a `dataset` specifically, not all the coins... If it's not the case then yes, it's going to fail everywhere else... 
As of right now I tested my tests with `FAMILY=tron yarn test path/to/all.libcore.js` and ignored the tests that were falling which were not mine cause they were related to cosmos mostly (I thought it was just a bug of the filtering method)... I launched a complete `yarn test` 2 hours ago pretty much and was waiting to see if it worked as expected 😖

Reply:
Yeah running the full `yarn test` is painful...
Some filters can be very handy, for example:

yarn test --runTestsByPath <one single test file, e.g. `src/__tests__/all.libcore.js`> --testNamePattern <name or description of a test, e.g `"balance is sum of ops"`>